java/test/jmri/jmrit/display/MemoryIconTest.java

```diff
@@ -91,6 +91,7 @@ protected void init() {
         Assert.assertNotNull("Instance exists", i);
         jmri.InstanceManager.memoryManagerInstance().provideMemory("IM2").setValue("");
         to.setMemory("IM2");
+        to.setMemory("IM2");
```

Comment:
More specifically, Pete added the duplicate setMemory in 1b731bbc481 14 days ago, and I fixed it in 5e949927 and bb529913ea1509.